### ISSUE TITLE
Support `phpstan-param` phpdoc

### ISSUE THREAD

**Issue by canvural** [klesun/deep-assoc-completion#177]
Looks like currently there is no support for the following phpdoc
```php
/** @phpstan-param array{foo:int, bar:string} $foo*/
function foo(array $foo)
{
	// ...
}
```

while `/** @param array{foo:int, bar:string} $foo*/` works.

Would be good to also treat `@phpstan-param` and `@psalm-param` same as `@param`

**Comment by klesun** [comment]
Hi. Thanks for the suggestion. `@psalm-param` should already work, but `@phpstan-param` support has to be added. Will do soon.

**Comment by klesun** [comment]
I take it should also include `@phpstan-return` and others:

https://phpstan.org/writing-php-code/phpdocs-basics

**Comment by canvural** [comment]
> `@psalm-param` should already work

Ah, good! Yeah I only tested it with `@phpstan-param` And assumed `@psalm-param` is also missing.

> Will do soon

Great! Thank you!

**Comment by klesun** [comment]
Released in [2021.03.11.001](https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/versions/stable/113448)

![image](https://user-images.githubusercontent.com/5202330/110788739-b12a7180-8277-11eb-9a23-2233fb0994a6.png)

(also added support for `@phan-*`)

**Comment by canvural** [comment]
Awesome! Thank you!
